### DISCUSSION TITLE
About the BCM94360NG WiFi card

### DISCUSSION THREAD

**Discussion by Zhonecage** [osy86/HaC-Mini#616] [General]
After reading [this thread](https://github.com/osy/HaC-Mini/issues/197), I bought the BCM94360NG wifi card, because it's supposed to work out-of-the-box according to many voices. Well it does work, but it has an issue that is a deal breaker for me: AirPods Pro compatibility. The Airpods connect just fine, but audio playback is very unstable, it cracks every couple seconds, like there is a bandwidth problem or something.

In comparison, the stock Intel card that ships with the Hades Canyon doesn't have this audio issue. It works after installing the Intel drivers, but it has another issue: Airdrop doesn't work. When I drop something from my iPhone to my Hades, it gets stuck at the end and the file is not sent.

BCM94360NG: Airpods Pro audio issues, Airdrop works
Stock Intel card: Airpods Pro work, Airdrop issues

Installing an original Apple Wifi card is out of the question for me because I need the second M.2 slot.
So yeah, all solutions are not ideal. Just letting you guys know in case you're undecided which wifi card to use.

**Comment by Medizinmann-MD** [comment]
I use a Wifi-Card with the BCM9460NG Chipset
Well - I don't have any Aipods to test.

But it works well with all bluetooth headphones I have tested like:
Jabra Elite 65t (true wireless in-ears)
Jabra Elite 75t (true wireless in-ears)
Beyerdynamic Amiron Wireless (over ear headset)

I also use an Apple Magic Trackpad 2  that works well.

WiFi is also okay.

That is all I can report.

I suspect that the Airpods don't connect with the right codec/bandwith  or it might be some kind of radio interference?

**Comment by Zhonecage** [comment]
It looks like the cracking audio is interference from the wifi. If I disable the wifi, the cracking goes away. BT and wifi both use 2.4Ghz, so there seems to be an issue with that. Strange that audio is fine using the Intel card despite wifi is on.

**Reply by Medizinmann-MD** [reply]
> It looks like the cracking audio is interference from the wifi. If I disable the wifi, the cracking goes away. BT and wifi both use 2.4Ghz, so there seems to be an issue with that. Strange that audio is fine using the Intel card despite wifi is on.

My card came with a sticker warning, that using 2.4GHz WiFi and Bluetooth simultaneously might lead to some interference.
I don’t use 2.4 GHz WiFi anymore – I changed everything to 5 GHz some time ago.
